packages/block-library/src/query-pagination-next/edit.js

```diff
@@ -12,6 +12,7 @@ export default function QueryPaginationNextEdit( {
 		<PlainText
 			__experimentalVersion={ 2 }
 			tagName="a"
+			style={ { display: 'inline-block' } }
```

Comment:
Not really related, but saw this when running full site editor e2e tests. Looks like the warnings are ignored so let's log an error instead.

https://github.com/WordPress/gutenberg/pull/30463/files#diff-cea529f9fa9034144e2e4f14ec3e87baa167522d3f3cda77b5ef904e47b817f0R18